client/src/List.svelte

```diff
@@ -238,6 +240,15 @@
 		});
 	}
 
+	function togglePaused() {
+		paused = !paused;
+		if (paused) {
+			if (interval) clearTimeout(interval);
+		} else {
+			event.set($event);
```

Comment:
Interesting; is this actually enough to trigger the whole update cycle, or will svelte realize that the value is the same as it was, and therefore do nothing?

Reply:
This does trigger the update- I think Svelte is of the opinion that the subscriber of a store should be responsible for ignoring duplicate values, which isn't the case for `event`.